packages/patternfly-4/react-table/src/components/Table/Body.tsx

```diff
@@ -116,7 +124,7 @@ export const TableBody = ({
   className = '' as string,
   children = null as React.ReactNode,
   rowKey = 'id' as string,
-  onRowClick = (...args: any) => undefined as any,
+  onRowClick = undefined,
```

Comment:
Is there any reason why you removed the empty function with undefined and `if`?

Reply:
Yes, I suppose it was two-fold. I replaced the default function because it executed every time a row was clicked, even if the table doesn't specify an `onRowClick` prop, which kinda looked like an error. Having two instances of `any` along with the type assertion `undefined as any` in the same function signature made me think it was just temporary implementation details that could stand to be cleaned up.